pom.xml

```diff
@@ -0,0 +1,25 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>juja.sqlcmd</groupId>
+    <artifactId>sqlcmd</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>sqlcmd</name>
+    <url>http://maven.apache.org</url>
```

Comment:
<url> is unnecessary tag that represents URL where the project lives. In this case, for example, you could use "https://github.com/DenysMelnychenko/sqlcmd"